src/python/gem5/resources/workload.py

```diff
@@ -148,7 +148,8 @@ class Workload(AbstractWorkload):
 
     # Optionally we can override a parameter in the workload. In this example
     # we are going to run this workload with a difference kernel.
-    workload.set_parameter("kernel", Resource("arm64-linux-kernel-4.14.134"))
+    workload.set_parameter("kernel", \
+        obtain_resource("arm64-linux-kernel-4.14.134"))
```

Comment:
I don't think the backslash is needed. I'm not sure how this got passed black...

```suggestion
    workload.set_parameter("kernel",
        obtain_resource("arm64-linux-kernel-4.14.134")
    )
```

Reply:
This code is in a docstring. It's an example of usage. Black probably just ignores it.

Regardless, I'll incorporate this.